Microsoft.Toolkit.Services/Services/MicrosoftTranslator/ServiceLanguage.cs

```diff
@@ -1,39 +1,42 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Newtonsoft.Json;
+
 namespace Microsoft.Toolkit.Services.MicrosoftTranslator
 {
     /// <summary>
     /// Holds information about langagues supported for text translation and speech synthesis.
     /// </summary>
+    /// <seealso cref="ITranslatorService.GetLanguageNamesAsync(string)"/>
     public class ServiceLanguage
     {
         /// <summary>
-        /// Gets the language code.
+        /// Gets or sets the language code.
         /// </summary>
-        public string Code { get; }
+        public string Code { get; set; }
```

Comment:
are the public setters necessary?

Reply:
At first time, I added the public setters because this class is now deserialized with JSON.NET, but actually they aren't necessary, because it is possible to use a class constructor to set these properties.
In fact, I have deleted the setters in other classes too (like `DetectedLanguage`, `Translation` and so on), and instead used a constructor to set their properties.